src/Uno.UI/UI/Xaml/Controls/Layouter/Layouter.cs

```diff
@@ -209,7 +211,7 @@ public void Arrange(Rect finalRect)
 
 				var arrangeSize = finalRect
 					.Size
-					.AtLeast(default); // 0.0,0.0
+					.AtLeastZero(); // 0.0,0.0
```

Comment:
Comment is a quite useless now :)

```suggestion
					.AtLeastZero();
```